spec.md

```diff
@@ -270,7 +270,19 @@ An asynchronous response triggers the platform marketplace to poll the endpoint
 
 #### Blocking Operations
 
-The marketplace MUST ensure that service brokers do not receive requests for an instance while an asynchronous operation is in progress. For example, if a broker is in the process of provisioning an instance asynchronously, the marketplace can not allow any update, bind, unbind, or deprovision requests to be made through the platform. A user who attempts to perform one of these actions while an operation is already in progress MUST receive a `400 Bad Request` response with the error message: `Another operation for this service instance is in progress`.
+This specification places no requirement on brokers with respect to their
```

Comment:
Right I was trying to say they're not required to support parallel requests and if they don't THIS is what they're supposed to return. I'll use your wording...